### DISCUSSION TITLE
Probably conflict

### DISCUSSION THREAD

**Discussion by SnailRabbids** [Poing-Studios/godot-admob-editor#48] [Q&A]
Hi guys, mb someone uses firebase-analytics plugin (or some another) and faced an issue, that AdMob plugin conflicts with another one?
How it was for me: (Xcode) builds ok -> installing to device -> Error -> Unable to install -> The code signature version is no longer supported

If I'll disable firebase-analytics plugin or admob plugin, then it works as expected.
(p.s. firebase-analytics uses pods and then project opens via .xcworkspace, not .xcodeproj, mb it is a problem)

![image](https://user-images.githubusercontent.com/36141552/184544298-652ee944-a1d8-442f-90ee-b2d539ca0995.png)

**Comment by gumaciel** [comment]
Hi can you provide us the link of this plugin? Yes, maybe can be conflict

**Reply by SnailRabbids** [reply]
@gumaciel 
Sure, here is: https://github.com/funseek/godot-ios-firebase-analytics
thanks :)

**Reply by gumaciel** [reply]
This plugins uses Cocoapods? I was wondering if we can also use Cocoapods, but I don't want to ask to user to convert the project to a Cocoapods project.

What you think about it? It's bad to ask to the user to convert the project to Cocoapods every time he needs to export the project?

**Reply by SnailRabbids** [reply]
I don't know, but this is the only firebase-analytics plugin I found that works :( so I have no choice and am forced to use Cocoapods.

**Reply by SnailRabbids** [reply]
Mb its possible to have plugin with pods and w/o?

**Reply by SnailRabbids** [reply]
I mean, 2 versions of same plugin (AdMob) with and without Cocoapods?

**Reply by gumaciel** [reply]
Yes, but I think it's not necessary 

The reason to use Cocoapods is this:

https://github.com/Poing-Studios/godot-admob-ios/issues/11

https://github.com/godotengine/godot-proposals/issues/4758

**Reply by SnailRabbids** [reply]
If it solves the conflict, then it should be done, I think
